knox-webshell-ui/webshell-ui/assets/.gitkeep

```diff
@@ -0,0 +1,17 @@
+##########################################################################
```

Comment:
Does this file needs to be checked in SVN?

Reply:
git wont track empty directory, this is to make sure asset folder is there for building purposes.

Reply:
got it.